src/filters/html.js

```diff
@@ -78,6 +82,7 @@ export class HTMLFilter {
             const translation = await translator(generateGuid(str), str);
             return translation && parseFragment(translation);
         });
-        return serialize(htmlAST);
+        return this.allowFragments && resource.indexOf('<html>')===-1? serialize(htmlAST).replace('<html><head></head><body>', '').replace('</body></html>', '') :
```

Comment:
what is the scenario where we want to disable `allowFragments`? if the source doesn't have an `<html>` tag should we ever want to generate it in output?

Reply:
Agree, we don't need the config. Removed.